src/Dexie/DexieStore.js

```diff
@@ -0,0 +1,66 @@
+//Dexie is not a typical database container, it's an abstraction on top of indexedDB which is already present in browsers  
+import Dexie from "dexie";
+
+export const db = new Dexie('User')
+
+db.version(1).stores({
+    symbols: "++symId, title, *symbols",
+    notes: "noteId, symId, title, content, date"
+})
+
+// db.symbols.bulkAdd([
+//     { title: 'Cynsies Tech', symbols: ["cyntech", "ctech"] },
+//     { title: 'DomsInd', symbols: ["doms"] },
+//     { title: 'AB real', symbols: ["adireal", "abreal"] },
+//     { title: 'Shiv Textile & Chemicals', symbols: ["shivtx", "shtexchem"] },
+//     { title: 'Kaka industries', symbols: ["kkind", 'kaka'] }
+// ]).then(() => console.log('done'))
+
+const dexieStore = {
+    getItem: getFunc,
+    setItem: setFunc,
+    db
+}
+
+async function getFunc(itemName) {
+    switch (itemName) {
+        case 'notes':
+            const notes = await db.notes.toArray()
+            return notes
+            break;
+
+        case 'currentNotes':
+            const { activeSymbol } = await chrome.storage.local.get(['activeSymbol']) || null
+            const currentNotes = await db.notes.where('symId').equals(activeSymbol.symId).toArray()
+            return currentNotes
+            break;
```

Comment:
can replace this with `return db.notes.where('symId').equals(activeSymbol.symId).toArray()`